index.html

```diff
@@ -440,6 +440,17 @@ <h5 class="uppercase mb0">My Journey with Google Code-In and FOSSASIA.</h5>
                                 </div>
 
                                 <div class="row">
+                    <div class="col-md-3 col-sm-6">
+						<a href="https://samyakbvs.wordpress.com/2017/12/07/just-go-with-the-flow/" target="_blank">
+							<div class="image-tile outer-title text-center">
+								<img src="img/blog/samyak.png" height="240" width="240">
```

Comment:
There's no image with name `satyam.png` I think your image is 6.jpg. please confirm and change accordingly.